kubert/Cargo.toml

```diff
@@ -108,6 +110,8 @@ futures-util = { version = "0.3", optional = true, default-features = false }
 hyper = { version = "0.14.17", optional = true, default-features = false }
 parking_lot = { version = "0.12", optional = true }
 pin-project-lite = { version = "0.2", optional = true }
+metrics-exporter-prometheus = { version = "0.11.0", optional = true, default-features = false}
+metrics-process = { version = "1.0.4", optional = true}
```

Comment:
perhaps the `metrics` dependencies ought to be feature flagged behind a `metrics` feature for Kubert?